changelog/17893.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:feature
```

Comment:
@zofskeez - Could you please open a PR to change this to the formatting for the "features" section of the changelog? Thanks!

Reply:
@mladlow oops I missed that in the readme! Sorry about that. #19062 updates the formatting.